lib/gringotts.ex

```diff
@@ -92,10 +101,8 @@ defmodule Gringotts do
         country: "US",
         postal_code: "11111"
       }
```

Comment:
payment map is been divided into `CreditCard` and `Address` struct. so @oyeb this example needs to be change accordingly.

Reply:
What's a payment map? I'm not familiar with the recent changes in stripe (and also AFK), so I'll appreciate it if you can push a commit to this branch with the corrections.

Reply:
@oyeb ok I will make the required changes.

Reply:
@zeus999 I don't understand how is this change flowing backwards from `gateway` to `gringotts.ex`, care to give more info.

Reply:
@pkrawat1 I think, the one who added the docs in `gringotts.ex` copied the docs from `stripe.ex`.

Reply:
@zeus999 I added that :1st_place_medal: anyways @oyeb please make sure this is fixed in a way such that it does not depend on the gateway! @oyeb @zeus999 collaborate on this.

Reply:
@pkrawat1  I am fixing this.